lighthouse-cli/types/types.ts

```diff
@@ -0,0 +1,37 @@
+interface AuditResult {
+  displayValue: string;
+  debugString: string;
+  comingSoon?: boolean;
+  score: number;
+  description: string;
+  extendedInfo?: {
```

Comment:
yeah I am purposely keeping the type definitions here only as specific as we need within the cli. If we were to pull these typedefs up into core I will be sure to fully match the reality of the objects